FastSimulation/Tracking/python/MixedTripletStep_cff.py

```diff
@@ -0,0 +1,86 @@
+import FWCore.ParameterSet.Config as cms
+
+# import the full tracking equivalent of this file
+import RecoTracker.IterativeTracking.MixedTripletStep_cff
+
+# trajectory seeds
+import FastSimulation.Tracking.TrajectorySeedProducer_cfi
+mixedTripletStepSeedsA = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.clone(
+    simTrackSelection = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.simTrackSelection.clone(
+        skipSimTrackIds = [
+            cms.InputTag("initialStepSimTrackIds"),
+            cms.InputTag("detachedTripletStepSimTrackIds"),
+            cms.InputTag("lowPtTripletStepSimTrackIds"),
+            cms.InputTag("pixelPairStepSimTrackIds")],
+        pTMin = 0.15,
+        maxD0 = 10.0,
+        maxZ0 = 30
+        ),
+    minLayersCrossed = 3,
+    ptMin =  RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsA.RegionFactoryPSet.RegionPSet.ptMin,
+    originRadius = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsA.RegionFactoryPSet.RegionPSet.originRadius,
+    originHalfLength = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsA.RegionFactoryPSet.RegionPSet.originHalfLength,
+    layerList = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedLayersA.layerList.value()
+)
+
+###
+import FastSimulation.Tracking.TrajectorySeedProducer_cfi
+mixedTripletStepSeedsB = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.clone(
+    simTrackSelection = FastSimulation.Tracking.TrajectorySeedProducer_cfi.trajectorySeedProducer.simTrackSelection.clone(
+        skipSimTrackIds = [
+            cms.InputTag("initialStepSimTrackIds"),
+            cms.InputTag("detachedTripletStepSimTrackIds"),
+            cms.InputTag("lowPtTripletStepSimTrackIds"),
+            cms.InputTag("pixelPairStepSimTrackIds")],
+        pTMin = 0.15,
+        maxD0 = 10.0,
+        maxZ0 = 30
+        ),
+    minLayersCrossed = 3,
+    ptMin =  RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsB.RegionFactoryPSet.RegionPSet.ptMin,
+    originRadius = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsB.RegionFactoryPSet.RegionPSet.originRadius,
+    originHalfLength = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedsB.RegionFactoryPSet.RegionPSet.originHalfLength,
+    layerList = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeedLayersB.layerList.value()
+)
+
+import RecoTracker.TkSeedGenerator.GlobalCombinedSeeds_cfi
+mixedTripletStepSeeds = RecoTracker.TkSeedGenerator.GlobalCombinedSeeds_cfi.globalCombinedSeeds.clone()
+mixedTripletStepSeeds.seedCollections = cms.VInputTag(
+    cms.InputTag('mixedTripletStepSeedsA'),
+    cms.InputTag('mixedTripletStepSeedsB'),
+    )
```

Comment:
@shchauha 

you can clone directly from full iterative tracking
mixedTripletStepSeeds = RecoTracker.IterativeTracking.MixedTripletStep_cff.mixedTripletStepSeed.Clone()

do the clone slightly different:

```
mixedTripletStepSeeds = RecoTracker.TkSeedGenerator.GlobalCombinedSeeds_cfi.globalCombinedSeeds.clone(
    mixedTripletStepSeeds.seedCollections = cms.VInputTag(
+    cms.InputTag('mixedTripletStepSeedsA'),
+    cms.InputTag('mixedTripletStepSeedsB'),
+    )
```